docs/mint.json

```diff
@@ -150,6 +150,7 @@
     {
       "group": "Changelog",
       "pages": [
+        "changelog/0.7.6",
```

Comment:
Thank you! If you ctrl-F for 0.7.5 in this file, you'll see the current version also needs to be upgraded to v0.7.6